### ISSUE TITLE
'\U' used without hex digits in character string starting "'C:\U"

### ISSUE THREAD

**Issue by jwijffels** [markvanderloo/tinytest#24]
when running on Windows

tinytest::build_install_test(getwd())

getting an error
here is the debug mode trace

```
Browse[2]> 
debug: scr <- sprintf(script, pkgname, tdir, testdir, pkgname, tdir, 
    at_home, verbose)
Browse[2]> 
debug: write(scr, file = "test.R")
Browse[2]> cat(scr)

suppressPackageStartupMessages({
  #        pkgname       tdir
  library('RMETAR', lib.loc='C:\Users\Jan\AppData\Local\Temp\RtmpcFy7Ee\file236c635c72e',character.only=TRUE)
  library('tinytest')
})
#                                testdir       pkgname       tdir          at_home     verbose
out <- run_test_dir(system.file('tinytest', package='RMETAR', lib.loc='C:\Users\Jan\AppData\Local\Temp\RtmpcFy7Ee\file236c635c72e'), at_home=TRUE, verbose=TRUE)
saveRDS(out, file='output.RDS')
Browse[2]> 
debug: system("Rscript test.R")
Browse[2]> 
Error: '\U' used without hex digits in character string starting "'C:\U"
Execution halted
debug: readRDS(file.path(tdir, "output.RDS"))
```

Need to replae \ with /?

**Comment by markvanderloo** [comment]
Thanks for reportning Jan! Reason is that the file path gets pasted with `sprintf` somewheres. Will fix.

(also, this demonstrates that you are the first Windows user to run it :))

**Comment by markvanderloo** [comment]
Fixed in 56bc96e. As a token of appreciation you have been immortalized in the NEWS file :). Nogmaals dank!

**Comment by jwijffels** [comment]
Dank je in naam van alle toekomstige windows gebruikers.
